docs/why-logfire/opentelemetry.md

```diff
@@ -0,0 +1,53 @@
+# OpenTelemetry under the hood :telescope:
+
+Because **Pydantic Logfire** is built on [OpenTelemetry](https://opentelemetry.io/), you can
+use a wealth of existing tooling and infrastructure, including
+[instrumentation for many common Python packages](https://opentelemetry-python-contrib.readthedocs.io/en/latest/index.html). Logfire also supports cross-language data integration and data export to any OpenTelemetry-compatible backend or proxy.
+
+For example, we can instrument a simple FastAPI app with just 2 lines of code:
+
+```py title="fastapi_example.py" hl_lines="8 9 10"
```

Comment:
All examples in the FastAPI docs use `main.py`, let's try to use the same notation.

```suggestion
```py title="main.py" hl_lines="8 9 10"
```

Reply:
done